pvlib/test/test_pvsystem.py

```diff
@@ -147,6 +147,28 @@ def test_PVSystem_physicaliam(mocker):
     pvsystem.physicaliam.assert_called_once_with(thetas, **module_parameters)
     assert iam < 1.
 
+def test__normalize_sam_product_names():
```

Comment:
two underscores in name, if that matters.

Reply:
One from the `test_` prefix and one from the function name.